metadata.md

```diff
@@ -167,6 +169,73 @@ The sample is a representative volume of wastewater taken from a site which is t
 
 -   **notes**: Any additional notes.
 
+## ContextualMeasurement (ContextualMeasurement.csv) <span id="ContextualMeasurement"><span>
+
+All sorts of measurements that are not performed on the wastewater sample but might provide additional context necessary for the interpretation of the results. 
+
+-   **ID**: (Primary Key) Unique identifier for each contextual measurement.
+
+-   **Site.ID**: (Foreign Key) Links with the Site table to describe the location of measurement.
+
+-   **dateTime**: The date and time the measurement was performed.
+
+-   **type**: The type of measurement that was performed	
+
+    -   `envTemperature`: Environmental temperature.
+    -   `envRainfall`: Amount of precipitation in the form of rain.
+    -   `envSnowFall`: Amount of precipitation in the form of snow.
+    -   `envSnowDepth`: Total depth of snow on the ground.
+    -   `wwFlow`: Flow of wastewater.
```

Comment:
Hi Doug,

The idea is to make a distinction between measurements performed on the sample used for Sars-Covid-19 (Measurement) and measurements NOT performed on the sample (contextualMeasurement). 

An example of each would be:

1. A measurement of the total suspended solids concentration of the composite sample taken specifically in the context of wastewater based surveillance for Sars-Covid-19. This involves a lab technician taking a sub-sample and filtering it after which the residue is heated and weighed. Clearly, such measurement results should link back to the sample, hence this option is still available in the Measurement table (`wqParamTSS`).

2. A daily average of the influent TSS concentration going into the WWTP. Clearly, this is not directly measured on the sample taken for Sars-Covid-19 (`wwTSS`). However, it might provide some background useful for future data analysis and modeling.

Do you agree?

Reply:
Yes and OK. But aren't measures such as the following going to be taken on a sample of the wastewater? 
    -   `wwTemp`: Temperature of the wastewater.
    -   `wwpH`: pH of the wastewater.
    -   `wwConductivity`: Conductivity of the wastewater.

We did discuss separating measures based on who and how the measures were performed. i.e. viral tests are performed in a lab and to group any wastewater tests that are performed by the same lab and people in that table (`Measurement` or could be renamed `viralMeasurement`. Separately group measures that are usually performed in the WWTP or site of collection or by different technicians or analysts. 

Are you following this approach/thinking?

Reply:
Hi Doug,

Regarding the first part of your comment. 
- Considering that the sample will be cooled and stored upon transportation, I don't think that temperature is very interesting to measure once the sample arrived in the lab.
- pH might be important though. 
- In Quebec, conductivity is usually measured to give an indication of snowmelt. In a single sample this doesn't make much sense, a daily average value is more appropriate in my opinion. I will discuss this further internally. 

For the second part of your comment, I'm not sure if I understand your thinking. Would it be possible to arrange a short online discussion soon? I'm available Tomorrow starting 7 am.